corona-warn-app/cwa-app-android
# Advanced root warning (EXPOSUREAPP-10698)

This PR adresses Expsoureapp-10698.

_As a CWA user using a rooted device I want to be able to hide the message about root detection to the next CWA upgrade, so that I don't get bothered every time by this message_

How to test:
- Take a rooted device
- check the message on app start
- disable the message by checkbox
- install app update

toDos:
- [x]  clean source
- [ ] add relevant testcases (maybe in follow up PR)


Screenshot:
![image](https://user-images.githubusercontent.com/46747780/146070009-d006a989-3dc4-4a41-a31a-8a4c5d0bbec9.png)
